### PR TITLE
Enable units test in Gluten*SQLInsertTestSuite

### PR DESCRIPTION
**Pull request by vibhaska**
## What changes were proposed in this pull request?
This PR enables following suites:
GlutenFileSourceSQLInsertTestSuite
GlutenDSV2SQLInsertTestSuite

(Fixes: \#3559)

## How was this patch tested?

Ran these UTs to confirm the fix

### CODE REVIEW

File: gluten-ut/spark34/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala [hunk 1]
```diff
@@ -1128,13 +1128,7 @@ class VeloxTestSettings extends BackendTestSettings {
   // following UT is removed in spark3.3.1
   // enableSuite[GlutenSimpleShowCreateTableSuite]
   enableSuite[GlutenFileSourceSQLInsertTestSuite]
-    // velox convert string null as -1583242847, which is not same with spark.
-    .exclude("SPARK-30844: static partition should also follow StoreAssignmentPolicy")
```
Comment (ayushi-agarwal) [inline gluten-ut/spark34/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala:1132]: This shouldn't have been removed

Comment (vibhaska) [reply gluten-ut/spark34/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala:1132]: Removed

**Comment by vibhaska** [comment]
@JkSelf Please review

**Review by JkSelf — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**